frontend/require/NavView.js

```diff
@@ -3,7 +3,7 @@ define(['underscore', 'backbone', 'mustache', 'text!NavView.html'], function(_,
 
     var NavView = Backbone.View.extend({
 
-        tagName: 'nav',
+        tagName: 'div',
```

Comment:
I don't understand why this is changed? When I tested, it doesn't seem to make any difference either way?